SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -31,15 +31,32 @@ define_repo_path() {
     local tmp_folder_name=$folder_name
 
     REPO_PATH="/tmp/${tmp_folder_name}"
+    GIT_REPO_PATH="${GIT_HOME}/${tmp_folder_name}"
 
     # Loop to find a unique folder name
-    while [ -d "${GIT_HOME}/${tmp_folder_name}" ]; do
-        counter=$((counter + 1))
-        tmp_folder_name="${folder_name}_${counter}"
-        REPO_PATH="/tmp/${tmp_folder_name}"
+    while [ -d "${GIT_REPO_PATH}" ]; do
```

Comment:
This deserves a comment that explains the logic. I.e.: we look for an already existing clone of the git repo. If we don't find any, the repository will be cloned again.